### PR TITLE
Add example to update passwordProfile of a user

### PR DESCRIPTION
**Pull request by FaithOmbongi**
Related to #13130 .
Adding an example to guide customers on how to reset a user password using the passwordProfile proeprty.

### CODE REVIEW

**Review by bpcastilho — APPROVED** [review]
It's great to have a sample.

File: api-reference/beta/api/user-update.md [hunk 1]
```diff
@@ -203,6 +203,37 @@ The following example shows the response.
 HTTP/1.1 204 No Content
 ```
 
+### Example 3: Update the passwordProfile of a user
```
Comment (dkershaw10) [inline api-reference/beta/api/user-update.md:206]: While this title is strictly correct, it's not scenario based.  Can we change this or add a sentence before the request to something like "Reset the password of another user"

**Review by dkershaw10 — APPROVED** [review]
Feel free to make a change or ignore my comment.  Either way, I'm happy with this improvement.

File: api-reference/beta/api/user-update.md (hunk 1 above)
Comment (FaithOmbongi) [reply api-reference/beta/api/user-update.md]: Totally acceptable & done.

**Pull request merged**